lib/Driver.js

```diff
@@ -60,7 +60,8 @@ module.exports = class Driver {
 
     let didDoc;
     try {
-      didDoc = await promise;
+      const {didDocument} = await promise;
+      didDoc = didDocument;
```

Comment:
```suggestion
      ({didDocument: didDoc} = await promise);
```

Reply:
Also, this cache stuff looks like a use case for: https://github.com/digitalbazaar/lru-memoize

Reply:
The cache stuff is stuff that was in here before... do we want to update it now?

Reply:
Upgraded to LRU memoization library in d36a78d0c931af47f563b0e18c3f91d9116d2a53.